source/_integrations/daikin.markdown

```diff
@@ -81,6 +82,10 @@ The `daikin` sensor platform integrates Daikin air conditioning systems into Hom
 Some models only report outside temperature when they are turned on.
 </div>
 
+<div class='note'>
+Some models does not have humidity sensor.
```

Comment:
Maybe it's better to merge the three notes to one that covers all?

Reply:
Maybe, I wanted to keep it separate as it's different information but if you like I'll merge it.

Reply:
I would suggest on a single one as well, wording suggestion:

```text
The reporting of power/energy consumption, humidity or outside temperature depends on the device model.
```